### PR TITLE
fix crash when passing invalid node id to droppeer and unban

### PR DESCRIPTION
**Pull request by vogel**
Crash fixed by catching exception.

Signed-off-by: Rafał Malinowski <rafal.przemyslaw.malinowski@gmail.com>

### CODE REVIEW

File: src/main/CommandHandler.cpp [hunk 1]
```diff
@@ -626,47 +626,54 @@ CommandHandler::connect(std::string const& params, std::string& retStr)
...
+        else
         {
-            retStr = "Peer ";
-            retStr += peerId->second;
-            retStr += " not found";
+            retStr = "Must specify at least peer id: droppeer?node=NODE_ID";
         }
     }
-    else
+    catch (std::exception& e)
```
Comment (jedmccaleb) [inline src/main/CommandHandler.cpp]: hmm where was it crashing?

Comment (vogel) [inline src/main/CommandHandler.cpp]: when passed node id was not starting with $ or @ and was not a valid nodeid (unable to parse)

Comment (jedmccaleb) [inline src/main/CommandHandler.cpp]: In resolveNodeID? We should just fix it there then

Comment (vogel) [inline src/main/CommandHandler.cpp]: I think that throwing an exception in this case is by design for resolveNodeID.

See old version of this method: https://github.com/stellar/stellar-core/blob/v0.5.0/src/main/Config.cpp#L817

It calls PubKeyUtils::fromStrKey: https://github.com/stellar/stellar-core/blob/v0.5.1/src/crypto/SecretKey.cpp#L269

which throws an exception.

If this is not desired behavior, then it requires a bigger change.

Comment (jedmccaleb) [inline src/main/CommandHandler.cpp]: Just a try catch in Config::resolveNodeID right? That returns false if it throws.

**Pull request merged**